src/community/mod.rs

```diff
@@ -67,6 +132,8 @@ pub struct Community {
     pub wiki1: Option<WikiPage>,
     pub wiki2: Option<WikiPage>,
     pub features: CommunityFeatureFlags,
+    pub addons: Vec<CommunityAddOn>,
+    pub configs: Vec<AddOnConfig>,
```

Comment:
There is no sense in splitting up these two. `CommunityAddOnConfig` should be, basically, a many-to-many connection between Community and AddOn entities.